SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -250,29 +193,6 @@ def customise_Validation_Trackingonly(process):
     return process
 
 def customise_harvesting(process):
-    # FIXME
-    #
-    # These should be added back once they get fixed
-    #
-    # The customizations are done here instead of in the central files
-    # with era because they are temporary
-
-    # Tracking DQM needs to be migrated for phase1
-    process.DQMHarvestTracking.remove(process.TrackingOfflineDQMClient)
-
-    # No HLT yet for 2017, so no need to run the DQM (avoiding excessive printouts)
-    process.DQMOffline_SecondStep.remove(process.HLTMonitoringClient)
-    process.DQMOffline_SecondStep_PrePOG.remove(process.hltOfflineDQMClient)
-    process.hltpostvalidation = cms.Sequence()
-    process.crt_dqmoffline.remove(process.dqmOfflineTriggerCert)
-
-    # Excessive printouts because 2017 doesn't have HLT yet
-    process.DQMOffline_SecondStep_PrePOG.remove(process.efficienciesRealData)
-    process.DQMOffline_SecondStep_PrePOG.remove(process.efficienciesRealElectronsData)
-    process.DQMOffline_SecondStep_PrePOG.remove(process.efficienciesRealMuonsData)
-    process.DQMOffline_SecondStep_PrePOG.remove(process.normalizePlotsRealMuonsData)
-    process.postValidation.remove(process.runTauEff)
-

```

Comment:
@schneiml I know the `sipixelCertification` is still in `dqmHarvesting`, but AFAICT it works technically fine without the input histograms (no crashes, no excessive printouts), which is why I left it there. Maybe enabling the phase1 pixel DQM makes it crash or something?

The module is defined here https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DQM/SiPixelCommon/python/SiPixelOfflineDQM_client_cff.py#L57 and here https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DQM/SiPixelCommon/python/SiPixelP5DQM_client_cff.py (I don't know what's the difference), and used here https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DQMOffline/Configuration/python/DQMOffline_CRT_cff.py#L18 and here 
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DQMOffline/Configuration/python/DQMOffline_Certification_cff.py#L13 (again, I don't know what's the difference).